cmd/controller-manager/app/controller_manager.go

```diff
@@ -303,7 +313,7 @@ func StartControllers(s *options.ControllerManagerServer,
 		glog.V(1).Info("Starting shared informers")
 		informerFactory.Start(stop)
 	} else {
-		glog.V(1).Infof("Skipping starting service-catalog controller because servicecatalog/v1alpha1 is not available")
+		return fmt.Errorf("unable to start service-catalog controller: servicecatalog/v1alpha1 is not available")
```

Comment:
setting up to write to the termination message path is probably worth doing, but is most likely a follow-up PR.